fission/function.go

```diff
@@ -276,3 +279,134 @@ func fnEdit(c *cli.Context) error {
 	fmt.Printf("function %v updated, new uuid: %v\n", newfn.Name, newfn.Uid)
 	return nil
 }
+
+func fnLogs(c *cli.Context) error {
+	client := getClient(c.GlobalString("server"))
+
+	fnName := c.String("name")
+	if len(fnName) == 0 {
+		fatal("Need name of function, use --name")
+	}
+
+	dbHost := c.String("dbhost")
```

Comment:
We should also all users to use env var here, instead of specifying this every time (also ok to do in a later change)